scripts/run-eslint.ts

```diff
@@ -44,7 +44,7 @@ main().catch((error) => {
 async function main() {
   const { cache, fix, quiet } = parseCommandLineArguments();
 
-  const eslint = new ESLint({ cache, fix });
```

Comment:
[The `fix` option instructs `lintFiles` to autofix errors as it finds them](https://eslint.org/docs/latest/integrate/nodejs-api#-new-eslintoptions). We manually call `outputFixes` in `runESLint` further down which does the same thing.